crates/matrix-sdk-ui/src/authentication/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod oidc;
+
+// Move AuthenticationService from the FFI into here.
```

Comment:
Left this as a placeholder for refactoring the `AuthenticationService` in the FFI into lightweight bindings and moving the main logic in here.